test/update-db/index.test.js

```diff
@@ -209,6 +209,16 @@ test('updates caniuse-lite for yarn', async () => {
   checkYarnLockfile(dir)
 })
 
+test('updates caniuse-lite for yarn without integrity', async () => {
+  let dir = await chdir(
+    'update-yarn-without-integrity',
```

Comment:
Added the test before the fix

- [x] Checked that it's failed
- [x] Check that it's resolved after the fix